tutorials/plot_background_filtering.py

```diff
@@ -366,8 +382,120 @@ def plot_signal(x, offset):
 plt.show()
 
 ###############################################################################
+# IIR filters
+# ===========
+# MNE-Python also offers IIR filtering functionality that is based on the
+# methods from :mod:`scipy.signal`. Specifically, we use the general-purpose
+# functions :func:`scipy.signal.iirfilter` and :func:`scipy.signal.iirdesign`,
+# which provide unified interfaces to IIR filter design.
+#
+# Designing IIR filters
+# ---------------------
+# Let's continue with our design of a 40 Hz low-pass filter, and look at
+# some trade-offs of different IIR filters.
+#
+# Often the default IIR filter is a `Butterworth filter`_, which is designed
+# to have a *maximally flat pass-band*. Let's look at a few orders of filter,
+# i.e., a few different number of coefficients used and therefore steepness
+# of the filter:
+
+sos = signal.iirfilter(2, f_p / nyq, btype='low', ftype='butter', output='sos')
+plot_filter(sos, 'Butterworth order=2', freq, gain)
+
+# Eventually this will just be from scipy signal.sosfiltfilt, but 0.18 is
+# not widely adopted yet (as of June 2016), so we use our wrapper...
+sosfiltfilt = mne.fixes.get_sosfiltfilt()
+x_shallow = sosfiltfilt(sos, x)
```

Comment:
can you add a line that says how do use it with raw data. Something like:

To use such a filter on raw data you can do:

```
iir_params = dict(ftype='butter', order=8, output='sos')
raw.filter(None, 40., method='iir', iir_params=iir_params)  # low pass filtering below 40 Hz
```